### ISSUE TITLE
a11Y writting change 

### ISSUE THREAD

**Issue by CyriaqueBillard** [Orange-OpenSource/a11y-guidelines#448]
I would like to suggest an evolution of the writing in the "Contraste des couleurs" paragraph in that page: https://a11y-guidelines.orange.com/fr/articles/couleur-et-taille-du-texte/
Currently, the writing distinguishes between "Texte standard", "Texte en gras". It make sense from the accessibility standard standpoint. But from a typography design standpoint, the writing is not correct. "Texte standard" lacks of clarity because in typography we differentiate by the weight. We can find Regular, Thin, Semi Bold, Bold, Extra Bold, etc... depending on the font.
"Texte standard" is not a weight. "Texte en gras" is a weight.
I suggest that we write "Texte standard (non gras)".

**Comment by pya35** [comment]
solved by PR #520
